binr/radare2/radare2.c

```diff
@@ -2,7 +2,11 @@
 
 #define USE_THREADS 1
 #define UNCOLORIZE_NONTTY 0
-
+#ifdef _MSC_VER
```

Comment:
what u mean? remove ifdef _MSC_VER or remove this and defines bellow?

Defines bellow are used to avoid missing headers form winsock into winbase.h/windows.h